laws/src/main/scala/cats/laws/discipline/Traverse1Tests.scala

```diff
@@ -0,0 +1,59 @@
+package cats.laws.discipline
+
+
+import org.scalacheck.{Arbitrary, Cogen, Prop}
+import Prop.forAll
+import cats.{Applicative, Eq, Monoid, Traverse1}
+import cats.laws.Traverse1Laws
+
+
+trait Traverse1Tests[F[_]] extends TraverseTests[F] with ReducibleTests[F] {
+  def laws: Traverse1Laws[F]
+
+  def traverse1[G[_]: Applicative, A: Arbitrary, B: Arbitrary, C: Arbitrary, M: Arbitrary, X[_]: Applicative, Y[_]: Applicative](implicit
+                                                                                                             ArbFA: Arbitrary[F[A]],
```

Comment:
I know indentation preferences are a bit personal (and we don't really have style guidelines), but this seems a bit excessive :smiley: 

If you base yourself `TraverseTests` and `TraverseLaws` as guidelines for indentation, `Traverse1Tests` and `Traverse1Laws` will be consistent with the rest.